model/strand.rb

```diff
@@ -57,11 +60,17 @@ def self.lease(id)
 
       yield
     ensure
-      num_updated = DB[<<SQL, id, lease].update
+      num_updated = DB[<<SQL, id, lease_time].update
 UPDATE strand
 SET lease = NULL
 WHERE id = ? AND lease = ?
 SQL
+
+      Clog.emit("lease cleared") do
+        {lease_cleared: {num_updated: num_updated}.tap {
+                          _1[:deleted] = @deleted if @deleted
```

Comment:
Why do we use it instead of `@deleted` or `!!@deleted`? Why does it return nil instead of false when `@deleted` is falsy?"

Reply:
Because I wanted to save space in the hash most of the time, when `@deleted` is nil or falsey, writing `deleted: false` continuously for something so common can be meaningful in log volumes.

But, after manipulating things in #700, there's not a good place for this code to go anymore, so it's gone.